### PR TITLE
Add upcoming plan name and pricing to current subscription page

### PR DESCRIPTION
**Pull request by TylerSheffels**
Refer to: http://manage.dimagi.com/default.asp?152666#850406

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/domain/views.py [hunk 1]
```diff
@@ -657,7 +659,12 @@ def plan(self):
                     next_subscription.update({
                         'exists': True,
                         'date_start': subscription.next_subscription.date_start.strftime("%d %B %Y"),
+                        'name': subscription.next_subscription.plan_version.plan.name,
+                        'price': self.get_product_summary(subscription.next_subscription.plan_version,
+                                                          self.account,
+                                                          subscription)[0]['monthly_fee'],
```
Comment (TylerSheffels) [inline corehq/apps/domain/views.py]: Not super proud of this, but I talked it through with @nickpell and we think there is never an instance when `get_product_summary()` returns more than 1 element.

Comment (nickpell) [inline corehq/apps/domain/views.py]: recommend flagging it in an error email with actionable text. example:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/accounting/models.py#L1106

Comment (czue) [inline corehq/apps/domain/views.py]: "we think there is never an instance when `get_product_summary()` returns more than 1 element" - could always put an `assert` for this if you wanted to fail hard in that scenario

**Comment by biyeun** [comment]
Can you add a screenshot?

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by TylerSheffels** [comment]
@nickpell @biyeun Should be ready to go.
image

**Pull request merged**